scipy/stats/setup.py

```diff
@@ -21,35 +23,39 @@ def configuration(parent_package='',top_path=None):
 
     # add statlib module
     config.add_extension('statlib',
-        sources=['statlib.pyf'],
-        f2py_options=['--no-wrap-functions'],
-        libraries=['statlib'],
-        depends=statlib_src
-    )
+                         sources=['statlib.pyf'],
+                         f2py_options=['--no-wrap-functions'],
+                         libraries=['statlib'],
+                         depends=statlib_src)
 
     # add _stats module
     config.add_extension('_stats',
-        sources=['_stats.c'],
-    )
+                         sources=['_stats.c'], )
 
     # add mvn module
     config.add_extension('mvn',
-        sources=['mvn.pyf', 'mvndst.f'],
-    )
+                         sources=['mvn.pyf', 'mvndst.f'], )
 
     # add _sobol module
     config.add_extension('_sobol',
-        sources=['_sobol.c', ],
-    )
+                         sources=['_sobol.c', ],
+                         )
```

Comment:
It would be nice to maintain a style consistent with your other changes and move this closing parenthesis to the end of the previous line.

Reply:
Same thanks